packages/grafana-data/src/utils/__snapshots__/csv.test.ts.snap

```diff
@@ -30,7 +30,7 @@ Object {
       "name": "Field 3",
       "type": "number",
       "values": Array [
-        null,
+        undefined,
```

Comment:
is this correct?

Reply:
We should choose what behavior we want, I go back and forth.

The question is what should be filled into the values that do not have anything configured.  To match the behavior of an array using an index that has nothing, I picked `undefined`, but previously I had explicitly filled up the missing values with `null`

Reply:
Think null is better, undefined is not valid json (for over the wire at least) and null is what Grafana normally use for missing data point